integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -9,15 +9,23 @@ set -e
 
 source /etc/os-release || source /usr/lib/os-release
 kubernetes_dir=$(dirname $0)
+KATA_HYPERVISOR="${KATA_HYPERVISOR:-qemu}"
 
-# Currently, Kubernetes tests only work on Ubuntu.
+# Currently, Kubernetes tests only work on Ubuntu and Centos.
 # We should delete this condition, when it works for other Distros.
-if [ "$ID" != ubuntu  ]; then
+if [ "$ID" != "ubuntu" ] && [ "$ID" != "centos" ]; then
```

Comment:
should this be an `or` instead of an `and`?

Reply:
@chavafg that will not work as it will fail in the first comparison... for example
```
 '[' centos '!=' ubuntu ']'
 echo 'Currently this script does not work on centos. Skipped Kubernetes Setup'
Currently this script does not work on fedora. Skipped Kubernetes Setup

```